src/Rules/Operators/OperandsInTernaryOperatorRule.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Operators;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Ternary;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+use PHPStan\Type\Constant\ConstantBooleanType;
+use PHPStan\Type\ConstantType;
+use PHPStan\Type\VerbosityLevel;
+
+/**
+ * @implements \PHPStan\Rules\Rule<\PhpParser\Node\Expr\Ternary>
+ */
+class OperandsInTernaryOperatorRule implements Rule
```

Comment:
The rule should be added to rules.neon similarly to this: https://github.com/phpstan/phpstan-phpunit/blob/1dd916d181b0539dea5cd37e91546afb8b107e17/rules.neon#L1-L7